CMakeLists.txt

```diff
@@ -108,15 +100,13 @@ add_subdirectory(cfile)
 add_subdirectory(czip)
 add_subdirectory(d3music)
 
-if(WIN32)
+if(CMAKE_SYTEM_NAME STREQUAL "Windows")
```

Comment:
Here typo that prevents build Windows, there should be `CMAKE_SYSTEM_NAME` (missed second "S" in SYSTEM).